specification/transport_layer/can.tex

```diff
@@ -33,26 +33,30 @@ \subsection{CAN ID structure}
             \multicolumn{4}{c|}{Service, not message} &
             \multicolumn{5}{c|}{Anonymous message} &
             \multicolumn{12}{c|}{\multirow{2}{*}{Subject-ID}} &
-            \multicolumn{8}{c|}{Protocol version}
-            \\\cline{2-4} \cline{7-10} \cline{23-29}
+            \multicolumn{1}{c|}{\multirow{2}{*}{R}} &
+            \multicolumn{7}{c|}{\multirow{2}{*}{Source node-ID}}
+            \\\cline{2-4} \cline{7-10}
 
             &
             \multicolumn{3}{c|}{Priority}
             &
             &
             &
-            \multicolumn{16}{c|}{} &
-            \multicolumn{7}{c|}{Source node-ID} &
+            R &
+            \multicolumn{15}{c|}{} &
+            &
+            \multicolumn{7}{c|}{}
             \\
 
             \textbf{Values} &
             \multicolumn{3}{c|}{$[0, 7]$} &
             $0$ &
             $\mathbb{B}$ &
-            \multicolumn{4}{c}{} &
+            $0$ &
+            \multicolumn{3}{c}{} &
             \multicolumn{12}{c|}{$[0, 32767]$} &
```

Comment:
Why was this listed at a 15-bit value before? Was this an error?

Reply:
The field was 16-bit wide but the range was limited to [0, 32767], implicitly making the most significant bit reserved at zero. Now its reservation is expressed explicitly.